utils/bin/lvim.template

```diff
@@ -1,9 +1,6 @@
 #!/usr/bin/env bash
 
-export LUNARVIM_RUNTIME_DIR="${LUNARVIM_RUNTIME_DIR:-RUNTIME_DIR_VAR}"
```

Comment:
Users have been using and expecting `LUNARVIM_xxx_DIR` variables to work for:
- customizing the installation path 
- have multiple installations
- have multiple configs (same base dir)
- maybe something else